metadata-ingestion/tests/integration/bigquery-usage/test_bigquery_usage.py

```diff
@@ -75,8 +75,8 @@ def test_bq_usage_source(pytestconfig, tmp_path):
         pipeline.run()
         pipeline.raise_from_status()
 
-    output = mce_helpers.load_json_file(str(tmp_path / "bigquery_usages.json"))
-    golden = mce_helpers.load_json_file(
-        str(test_resources_dir / "bigquery_usages_golden.json")
+    mce_helpers.check_golden_file(
+        pytestconfig,
+        output_path=tmp_path / "bigquery_usages.json",
```

Comment:
Is there a reason why str() was removed here while being retained in other places? Could we move it inside of `check_golden_file`?

Reply:
No strong reason - I'll make it consistent